lib/content-services/src/lib/content-metadata/components/content-metadata/button-type.enum.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// eslint-disable-next-line no-shadow
+export enum ButtonType {
```

Comment:
Did you test this approach with some custom types of data displayed in properties panel such as record properties?

Reply:
The record card is displayed when we select 'declare record' from the context menu. This functionality is working fine as earlier. Please check the recording & screenshot.
[screen-capture (1).webm](https://github.com/Alfresco/alfresco-ng2-components/assets/111955585/9edf696e-9121-4035-b218-2959e37adb3b)
![Screenshot from 2023-09-29 18-09-38](https://github.com/Alfresco/alfresco-ng2-components/assets/111955585/ea0a9adf-67b6-4d94-b7fd-a1e55717aced)

Reply:
It is working fine as earlier. Please check the recording and screenshot.
[screen-capture (1).webm](https://github.com/Alfresco/alfresco-ng2-components/assets/111955585/0e1d28e0-a4c2-4083-bca2-9491b8d7ce11)
![Screenshot from 2023-09-29 18-09-38](https://github.com/Alfresco/alfresco-ng2-components/assets/111955585/7f5e790e-4289-4ee1-960f-9ea3645ab464)

Reply:
Okay, thanks for checking this out